### ISSUE TITLE
Summarise feature survey results so far

### ISSUE THREAD

**Issue by manics** [sa-tre/satre-team#4]
### Summary of issue

We have some initial responses from our TRE features survey
https://dundee.onlinesurveys.ac.uk/satre-tre-operatorsbuilders-survey

Create an initial summary chart of responses

### What needs to be done?

- Jupyter notebook with summary chart of responses: one bar chart for each question that asks for a rating

### Who can help?

- @manics

### Issue checklist

- [X] I have assigned the appropriate work package label to this issue, and added any other relevant labels
- [X] I have added this to the `SATRE backlog (public)` project board
- [X] I have assigned this issue to at least one person on the SATRE team

**Comment by JimMadge** [comment]
This made me think. Have we thought about presenting the final results (or at least how we generated them for any publications) in some literate programming manner?

While we can't share the raw data to let people reproduce the results (and we would need to be careful not to reveal anything we cannot share), it might still be helpful in demonstrating how we reached our conclusions and give confidence about how we handled the data.
